wcfsetup/install/files/lib/data/user/UserProfile.class.php

```diff
@@ -463,7 +469,11 @@ public function isAccessible($name) {
 	 * @return	boolean
 	 */
 	public function isProtected() {
-		return (!WCF::getSession()->getPermission('admin.general.canViewPrivateUserOptions') && !$this->isAccessible('canViewProfile') && $this->userID != WCF::getUser()->userID);
```

Comment:
This change is not necessary, `isAccessible()` already offers the ability for event listeners to alter the result. Revert the changes to this particular method please.